target/linux/generic/config-5.15

```diff
@@ -1360,8 +1375,6 @@ CONFIG_DEFCONFIG_LIST="/lib/modules/$UNAME_RELEASE/.config"
 # CONFIG_DEVMEM is not set
 CONFIG_DEVPORT=y
 # CONFIG_DEVPTS_MULTIPLE_INSTANCES is not set
-# CONFIG_DEVTMPFS is not set
```

Comment:
I would leave the symbol here instead of disabling it in targets. This difference was noticed by me while refreshing config for armvirt to test kernel 5.15.